### PR TITLE
Standalone helpdesk installation configuration

### PR DESCRIPTION
**Pull request by timthelion**
Fixes https://github.com/django-helpdesk/django-helpdesk/issues/826

You can easily configure themes and alternate email senders such as SES. This is documented in the new standalone Readme.

### CODE REVIEW

**Comment by timthelion** [comment]
I see another helpful contributor was working on this at the same time I was. I believe my version is better though because it has more production ready features such as using gunicorn as well as support for receiving email using a cron job...

https://github.com/django-helpdesk/django-helpdesk/pull/1117/files

**Comment by Benbb96** [comment]
I didn't have time to review it but I hope I'll be able to do it soon..

Maybe you can also update the README like on the other PR, in order to show the commands to use the docker-compose ?

File: standalone/Caddyfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+localhost {
+        reverse_proxy helpdesk:8000
```
Comment (Benbb96) [inline standalone/Caddyfile:2]:
```suggestion
        reverse_proxy django-helpdesk:8000
```
I had to change this to make it match the container name in the docker-compose file to make it work

File: standalone/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ django-helpdesk:
+   build:
+    context: ..
+    dockerfile: standalone/Dockerfile
+   user: root
+   volumes:
+    - /tmp/django-helpdesk-data:/data/
+    - ./custom_navigation_header.html:/opt/django-helpdesk/helpdesk/templates/helpdesk/custom_navigation_header.html:r
+   env_file: docker.env
+
```
Comment (Benbb96) [inline standalone/docker-compose.yml:23]:
```suggestion
   env_file: docker.env
   depends_on:
     - postgres

```
In order to be sure db is ready to run migrations

**Review by Benbb96 — COMMENTED** [review]
I encountered some errors while trying on Windows (setup.sh used wrong end of line but that could be due of PyCharm), then I tested on WSL Debian and it worked (after fixing Caddyfile).

It seems good for me. Only media files didn't work for me. But I didn't dig too much to make them work.

**Comment by timthelion** [comment]
What did you have to do to fix the Caddyfile?

File: standalone/Caddyfile (hunk 1 above)
Comment (Benbb96) [reply standalone/Caddyfile]: @timthelion Here I had to make it match the name of the container from the `docker-compose.yml`

**Review by Benbb96 — APPROVED** [review]

File: helpdesk/models.py [hunk 3]
```diff
@@ -207,7 +207,7 @@ class Queue(models.Model):
     email_box_ssl = models.BooleanField(
         _('Use SSL for E-Mail?'),
         blank=True,
-        default=False,
+        default=True,
```
[the remark this answers is not in the document]
Comment (uhurusurfa) [reply helpdesk/models.py:210]: This probably will require a migration.

**Comment by uhurusurfa** [comment]
I have a feling the expunge call is messing with the unit tests.

**Comment by timthelion** [comment]
Interesting thought, just reverted that to see if it helps...

**Comment by timthelion** [comment]
Turns out it was that bad default.
We should still change the default though...

**Comment by timthelion** [comment]
But in another PR

**Comment by timthelion** [comment]
I don't know if you noticed but I also added a crappy icon ;) So now we have a logo...

**Review by Benbb96 — APPROVED** [review]

**Pull request merged**